src/Entity/User.php

```diff
@@ -68,6 +69,11 @@ public function setEmail(string $email): static
         return $this;
     }
 
+    public function getEmailAddress(): Address
+    {
+        return new Address($this->email, $this->name);
+    }
```

Comment:
I'm a bit "meh" on this: we use it in one place & the user may call it expecting the string